### PR TITLE
[MER-802] Update Google SSO button to comply with Google requirements

### PR DESCRIPTION
**Pull request by gastonabella**
[MER-802](https://eliterate.atlassian.net/browse/MER-802)

Update auth providers styling in order to comply with [Google requirements](https://developers.google.com/identity/branding-guidelines#top_of_page).

### Before:
Screen Shot 2022-02-22 at 11 39 53 AM

### After:
Screen Shot 2022-02-22 at 11 42 41 AM
Screen Shot 2022-02-22 at 11 42 59 AM

### CODE REVIEW

**Review by jrissler — APPROVED** [review]

**Comment by jrissler** [comment]
I approved with a pending question: is it worth adding any test for the pow stuff? If not, ignore - if so, could be good as long as it isn't solely testing pow (that should be done in their code, not ours).

File: assets/styles/common/box-form-container.scss [hunk 1]
```diff
@@ -129,23 +126,63 @@
...
+      background-color: darken($google-button-background-color, 3%) !important;
+    }
+  }
+
+  .google-auth-container{
+    display: flex;
+    align-items: center;
+
+    .google-icon-container{
+      padding-right: 6px;
```
Comment (mcass19) [inline assets/styles/common/box-form-container.scss]: Is there a possibility of being consistent and use `em` and `rem` always? (maybe it's too hard to change)

**Review by mcass19 — APPROVED** [review]

**Comment by mcass19** [comment]
> I approved with a pending question: is it worth adding any test for the pow stuff? If not, ignore - if so, could be good as long as it isn't solely testing pow (that should be done in their code, not ours).

We have [this test](https://github.com/Simon-Initiative/oli-torus/blob/master/test/oli_web/pow/pow_test.exs) that maybe we can add something there? Or what idea did you have in mind? @jrissler

**Comment by jrissler** [comment]
Yeah something along those lines - if it is worth it (if not 🤷 )

**Comment by gastonabella** [comment]
@jrissler @mcass19 I've just pushed the changes with your suggestions. I added some pow tests to ensure the Sign In buttons for providers are present. Thanks!

**Comment by mcass19** [comment]
Awesome! Good to go from my side 🚀 🚀 🚀

**Pull request closed without merging**